addons/website/static/src/js/menu/edit.js

```diff
@@ -189,6 +189,8 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
         this.trigger_up('widgets_stop_request', {
             $target: this._targetForEdition(),
         });
+        // Restore dropdown behavior
+        $('.dropdown-toggle').unbind().attr('data-toggle', 'dropdown').dropdown('hide');
```

Comment:
- http://api.jquery.com/unbind/#entry-longdesc
- Also since you bind an event on starting the `Wysiwyg` multizone... unbind the event on destroying the `Wysiwyg` multizone, so more than probably not here. This may work like that but this is "spaguetti" code, you add an half feature somewhere, then the other half at a position which works... but which is not consistent. Someone looking at `WysiwygMultizone` will see that you bind the event but do not seem to unbind it ever... why would he think that you did it here ?